### PR TITLE
[BAEL-4173] Set JWT token with Spring Boot & Swagger UI

### PR DESCRIPTION
**Pull request by saleco**

### CODE REVIEW

File: spring-boot-modules/spring-boot-swagger-jwt/src/main/java/com/baeldung/swaggerjwt/controller/ClientsRestController.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import io.swagger.annotations.ApiOperation;
+
+@RestController(value = "/clients")
+@Api( tags = "Clients")
+public class ClientsRestController {
+
+    @ApiOperation(value = "This method is used to get the clients.")
+    @GetMapping
+    public List<String> getClients(@RequestHeader HttpHeaders httpHeaders) {
+        System.out.println("JWT TOKEN: " + httpHeaders.getFirst(SwaggerConfiguration.AUTHORIZATION_HEADER));
```
Comment (KevinGilmore) [inline spring-boot-modules/spring-boot-swagger-jwt/src/main/java/com/baeldung/swaggerjwt/controller/ClientsRestController.java:22]: Let's avoid the println.

**Review by KevinGilmore — CHANGES_REQUESTED** [review]
There a few things to adjust. Also, if it's possible for this code to coexist in the existing spring-boot-swagger module, then we should move it there instead of creating a new module.

**Comment by saleco** [comment]
> There a few things to adjust. Also, if it's possible for this code to coexist in the existing spring-boot-swagger module, then we should move it there instead of creating a new module.

Hello Kevin, not sure its possible to coexist with the existing module since I will change some of the configurations and its not possible to have two swagger configurations in the same project.

File: spring-boot-modules/spring-boot-swagger-jwt/src/main/java/com/baeldung/swaggerjwt/configuration/SwaggerConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+@Configuration
+public class SwaggerConfiguration {
+
+    public static final String AUTHORIZATION_HEADER = "Authorization";
+
+    private ApiInfo apiInfo() {
+        return new ApiInfo("My REST API", "Some custom description of API.", "1.0", "Terms of service",
+          new Contact("Sallo Szrajbman", "www.baeldung.com", "salloszraj@gmail.com"),
+          "License of API", "API license URL", Collections.emptyList());
```
[the remark this answers is not in the document]
Comment (saleco) [reply spring-boot-modules/spring-boot-swagger-jwt/src/main/java/com/baeldung/swaggerjwt/configuration/SwaggerConfiguration.java]: Hello kevin, I updated the identadion... Not sure I did it correctly... Could you please check in the last commit?

File: spring-boot-modules/spring-boot-swagger-jwt/src/main/java/com/baeldung/swaggerjwt/configuration/SwaggerConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    public static final String AUTHORIZATION_HEADER = "Authorization";
+
+    private ApiInfo apiInfo() {
+        return new ApiInfo("My REST API",
+		  "Some custom description of API.",
+		  "1.0",
+		  "Terms of service",
+		  new Contact("Sallo Szrajbman",
+		    "www.baeldung.com",
+		    "salloszraj@gmail.com"),
```
Comment (KevinGilmore) [inline spring-boot-modules/spring-boot-swagger-jwt/src/main/java/com/baeldung/swaggerjwt/configuration/SwaggerConfiguration.java:28]: The new Contact(...) can still be on one line; it's just the lengthy list of parameters passed to the ApiInfo constructor that won't fit on one line and need to be broken up.

Comment (saleco) [reply spring-boot-modules/spring-boot-swagger-jwt/src/main/java/com/baeldung/swaggerjwt/configuration/SwaggerConfiguration.java]: Thanks for the heads up! It's now fixed.

**Review by KevinGilmore — APPROVED** [review]

File: spring-boot-modules/spring-boot-swagger-jwt/src/main/java/com/baeldung/swaggerjwt/controller/ClientsRestController.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import io.swagger.annotations.ApiOperation;
+
+@RestController(value = "/clients")
+@Api( tags = "Clients")
+public class ClientsRestController {
+
+    @ApiOperation(value = "This method is used to get the clients.")
+    @GetMapping
+    public List<String> getClients() {
+        return Arrays.asList("Fisrt Client", "Second Client");
```
Comment (KevinGilmore) [inline spring-boot-modules/spring-boot-swagger-jwt/src/main/java/com/baeldung/swaggerjwt/controller/ClientsRestController.java:22]:
```suggestion
        return Arrays.asList("First Client", "Second Client");
```

**Review by KevinGilmore — CHANGES_REQUESTED** [review]
Correct spelling of "First"

**Review by KevinGilmore — APPROVED** [review]

**Pull request merged**